src/neml2/models/solid_mechanics/MixedControlSetup.cxx

```diff
@@ -42,7 +43,7 @@ MixedControlSetup::expected_options()
       "The name of the control signal.  Values less than the threshold are "
       "strain control, greater are stress control";
 
-  options.set<Real>("threshold") = 0.5;
+  options.set<BatchTensor>("threshold") = BatchTensor::full({}, 0.5);
```

Comment:
This is probably leading to the doco failure as the default value serializes to a multi line string... I can try to fix it when I get back unless you find a workaround before I do.